### PR TITLE
feat(vector): adding aerial imagery style json

### PR DESCRIPTION
**Pull request by blacha**

### CODE REVIEW

File: config/style/aerial.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      "tileSize": 256,
+      "tiles": ["/v1/tiles/aerial/EPSG:3857/{z}/{x}/{y}.webp"]
+    }
+  },
+  "layers": [
+    {
+      "id": "Aerial-Imagery",
+      "type": "raster",
+      "source": "LINZ-Imagery",
+      "layout": { "visibility": "visible" }
```
Comment (Wentao-Kuang) [inline config/style/aerial.json:17]: layout seems not necessary here?

**Review by Wentao-Kuang — APPROVED** [review]
Looks good to me, we also need to update the landing page `TileGrid.getStyle()` to be able to test this. And also need an `s=aerial` as default.

File: config/style/aerial.json (hunk 1 above)
Comment (blacha) [reply config/style/aerial.json:17]: This was just a copy paste from somewhere else, but yeah lets just take this out.

**Review by Wentao-Kuang — APPROVED** [review]

**Comment by Wentao-Kuang** [comment]
Another concern is that this going to limit our aerial only limited to webp format. We also need do some works in landing page to change the source url.

**Comment by blacha** [comment]
> Another concern is that this going to limit our aerial only limited to webp format. We also need do some works in landing page to change the source url.

Yes and also EPSG:3857

I have been wondering about this, we sort of need a template system `"/v1/tiles/aerial/{projection}/{z}/{x}/{y}.{format}?api={api-token}"`

**Comment by Wentao-Kuang** [comment]
> 

That's exactly what I am thinking. Even we can add an {HOST} at the beginning.

**Comment by blacha** [comment]
This will be configured as a virtual tileset inside the import process.

**Pull request closed without merging**